feature-detects/mathml.js

```diff
@@ -21,8 +21,8 @@ define(['Modernizr', 'testStyles'], function( Modernizr, testStyles ) {
   Modernizr.addTest('mathml', function() {
     var ret;
 
-    testStyles('#modernizr{position:absolute}', function(node){
-      node.innerHTML = '<math><mfrac><mi>xx</mi><mi>yy</mi></mfrac></math>';
+    testStyles('#modernizr{position:absolute;display:inline-block}', function(node){
```

Comment:
yesir. (I presume) the detect came out of [this conversation](https://github.com/mathjax/MathJax/issues/182). The issue here is that we are injecting into a div with block styling. so depending on the styling of the page, the resulting div will be much wider than the height. On my testing page on FF35 via a MBPr, the height was 30px, and the width was 960px.

I actually think just checking the height of the element would be sufficient, but I imagine that the fine people at MathJax know much more than I do about support.